archetypes/helidon/src/main/archetype/common/observability.xml

```diff
@@ -249,58 +298,50 @@ curl -H 'Accept: application/json' -X GET http://localhost:8080/metrics
 
         assertThat(message, is("Hello Joe"));
         Counter counter = registry.counter("personalizedGets");
-        double before = counter.count();
+        double before = counter.getCount();
 
         message = target.path("simple-greet/Eric")
                 .request()
                 .get(String.class);
 
         assertThat(message, is("Hello Eric"));
-        double after = counter.count();
+        double after = counter.getCount();
         assertEquals(1d, after - before, "Difference in personalized greeting counter between successive calls");
     }]]></value>
                                     </list>
-                                    <list key="MainTest-static-fields" if="${flavor} == 'mp'">
-                                        <value><![CDATA[
-    @Inject
-    private MeterRegistry registry;]]></value>
```

Comment:
I am not sure what code used the injected `MeterRegistry` before these changes. (Maybe none?) Certainly we should not inject a `MeterRegistry` but injecting the MP `MetricRegistry` and then using it is fine for an MP app. Certainly if the code would not use an injected MP `MetricRegistry` then there's no need to inject it.

Reply:
I changed `MeterRegistry` to `MetricRegistry` for MP application. You can see this code few lines above:
```xml
<list key="MainTest-static-fields" if="${flavor} == 'mp'">
                                        <value><![CDATA[
    @Inject
    private MetricRegistry registry;]]></value>
</list>
```
But as I moved it, it looks like I removed it but I just modified it.